site/zh-cn/tutorials/text/transformer.ipynb

```diff
@@ -0,0 +1,2057 @@
+        "                                  (batch_size, -1, self.d_model))  # (batch_size, seq_len_q, d_model)\n",
+        "\n",
+        "    output = self.dense(concat_attention)  # (batch_size, seq_len_q, d_model)\n",
+        "        \n",
+        "    return output, attention_weights"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "0D8FJue5lDyZ"
+      },
+      "source": [
+        "创建一个 `MultiHeadAttention` 层进行尝试。在序列中的每个位置 `y`，`MultiHeadAttention` 在序列中的所有其他位置运行所有8个注意力头，在每个位置y，返回一个新的同样长度的向量。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "Hu94p-_-2_BX",
+        "colab": {}
+      },
+      "source": [
+        "temp_mha = MultiHeadAttention(d_model=512, num_heads=8)\n",
+        "y = tf.random.uniform((1, 60, 512))  # (batch_size, encoder_sequence, d_model)\n",
+        "out, attn = temp_mha(y, k=y, q=y, mask=None)\n",
+        "out.shape, attn.shape"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "RdDqGayx67vv"
+      },
+      "source": [
+        "### 点式前馈网络（Point wise feed forward network）"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "gBqzJXGfHK3X"
+      },
+      "source": [
+        "点式前馈网络由两层全联接层组成，两层之间有一个 ReLU 激活函数。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "ET7xLt0yCT6Z",
+        "colab": {}
+      },
+      "source": [
+        "def point_wise_feed_forward_network(d_model, dff):\n",
+        "  return tf.keras.Sequential([\n",
+        "      tf.keras.layers.Dense(dff, activation='relu'),  # (batch_size, seq_len, dff)\n",
+        "      tf.keras.layers.Dense(d_model)  # (batch_size, seq_len, d_model)\n",
+        "  ])"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "mytb1lPyOHLB",
+        "colab": {}
+      },
+      "source": [
+        "sample_ffn = point_wise_feed_forward_network(512, 2048)\n",
+        "sample_ffn(tf.random.uniform((64, 50, 512))).shape"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "7e7hKcxn6-zd"
+      },
+      "source": [
+        "## 编码与解码（Encoder and decoder）"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "yScbC0MUH8dS"
+      },
+      "source": [
+        "<img src=\"https://www.tensorflow.org/images/tutorials/transformer/transformer.png\" width=\"600\" alt=\"transformer\">"
```

Comment:
```suggestion
        "<img src=\"https://tensorflow.google.cn/images/tutorials/transformer/transformer.png" width=\"600\" alt=\"transformer\">"
```

Reply:
Thanks for the review, the links are changed in the latest commit.